edx-platform/xblocks/config.yaml

```diff
@@ -0,0 +1,21 @@
+xblocks:
+#  - name: xblock-drag-and-drop-v2
+#    upstream_repo: https://github.com/edx-solutions/xblock-drag-and-drop-v2.git
+#    local_repo: git@github.com:appsembler/xblock-drag-and-drop-v2.git
+#    requirement_steps:
+#      - pip install -r requirements.txt
+#    extract_steps:
+#      - make extract_translations
+#      - make push_translations
+#    pull_steps:
+#      - make pull_translations
+
+  - name: xblock-poll
+    upstream_repo: https://github.com/open-craft/xblock-poll.git
+    local_repo: git@github.com:appsembler/xblock-poll.git
+    requirement_steps:
+      - make requirements
+    push_steps:
+      - make push_translations
+    pull_steps:
+      - make pull_translations
```

Comment:
This installs all requirements on the host, and it requires us to define a DSL and a parser for this language (as you did in xblocks.py). As an alternative, may I suggest that we create a different docker image for each xblock? This will allow us to have cleanly-separated environments for each xblock. Also, we won't have to define a DSL, as we will just have to run `docker run openedx-i18n-xblock-poll` to get the po files.

Reply:
The DSL issue, we can find a better alternative, no problem.

There are soo many XBlocks, we can't just make an image for every one of them. Plus it's using a throw away Open edX platform Docker image, so it's not making anything dirty. It's heavy, yes. But it's not an optimization that I would worry about now.

What do you think?

Reply:
Well, we're going to have to write custom installation instructions for every xblock, so I don't see much difference with writing a custom Dockerfile.

But I don't want to block this PR and waste your time because of this. Just tell me when you are ready to merge.

Reply:
So yeah. I get your point @regisb and it's definitively correct. Having multiple repositories is a challenge to manage. Especially when it's not just code, but also translations.

I'm sure you know that JavaScript translation are hardcoded within the XBlock and the platform has no influence on those translations.

So, I'm trying to find the laziest way to make this happen without spending a full day updating 10 requirements just to get the latest requirements.

Reply:
Lazy is good! Let me know when you're ready to merge.